### ISSUE TITLE
Performance of Semantic Segmentation on Pascal VOC

### ISSUE THREAD

**Issue by WesleyHsieh0806** [WXinlong/DenseCL#19]
Hi, I tried to reproduce your results on VOC Semantic Segmentation, but only got **mIOU = 46.87** (while you can achieve **69.4**)
Can you give me some help?

Here's the steps I have done for reproduction.

1.  Download your Pretrained model [DenseCL IN/200Epoch](https://cloudstor.aarnet.edu.au/plus/s/hdAg5RYm8NNM2QP/download)
2.  Follow your steps in [Readme.md](https://github.com/WXinlong/DenseCL/blob/main/benchmarks/detection/README.md)

I did not modify any setting about batch size or learning rate.
Is there anything I have ignored?

**Comment by lcysonya** [comment]
@WesleyHsieh0806  I got the same issue, trained VOC2012(no aug) semantic segmentation from COCO 800ep pretrain and only got mIoU=45.48

Did you solve the problem? 

@WXinlong What should I do to get the results reported?
